.github/workflows/rear_panel.yaml

```diff
@@ -52,7 +52,7 @@ jobs:
           path: "./stm32-tools"
           key: ${{ runner.os }}-${{ hashFiles('**/cmake/*') }}-${{ secrets.CACHE_VERSION }}
       - name: "Configure"
-        run: cmake --preset=cross . -DCMAKE_BUILD_TYPE=debug
```

Comment:
not sure why you made this change, but the history of specifying the build type as debug is that sometimes the optimized code will suppress some actual errors in code.

Reply:
We changed all of the boards to debug because the cross compiled debug builds no longer fit in the ROM of the chips.